sdk/core/azure-core/src/main/java/com/azure/core/implementation/SwaggerMethodParser.java

```diff
@@ -53,6 +54,7 @@
  * method.
  */
 public class SwaggerMethodParser implements HttpResponseDecodeData {
+    private ClientLogger logger = new ClientLogger(SwaggerMethodParser.class);
```

Comment:
private final. Unless you plan on reassigning this somewhere else in the class.

Reply:
Done.